subprojects/gradle/room-config/src/main/kotlin/com/avito/android/plugin/incremental/SystemOutMessenger.kt

```diff
@@ -0,0 +1,31 @@
+package com.avito.android.plugin.incremental
+
+import javax.annotation.processing.Messager
+import javax.lang.model.element.AnnotationMirror
+import javax.lang.model.element.AnnotationValue
+import javax.lang.model.element.Element
+import javax.tools.Diagnostic
+
+internal class SystemOutMessenger : Messager {
```

Comment:
It's hard to understand what is going on

Reply:
If we need a stub I think, it should simply do nothing

Reply:
Not actual. Move this check from this PR to another task.